blog/20131108-debian-mate-packaging-team.md

```diff
@@ -8,21 +8,21 @@
 .. author: Stefano Karapetsas
 -->
 
-![Debian](http://www.debian.org/logos/openlogo-nd-100.png)
+![Debian](https://www.debian.org/logos/openlogo-nd-100.png)
 
 The MATE Team is very happy to say hello to the new Debian MATE Packaging
-Team, that is [working hard](http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=708385#31)
+Team, that is [working hard](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=708385#31)
 to get MATE included into the next release of Debian.
 
 First packages are already in the repositories and there are many others
-in [ftp-master NEW queue](http://ftp-master.debian.org/new.html).
+in [ftp-master NEW queue](https://ftp-master.debian.org/new.html).
 
 A big thanks to:
 
-  * [John Paul Adrian Glaubitz](http://users.physik.fu-berlin.de/~glaubitz/)
-  * [Mike Gabriel](http://sunweavers.net/blog/)
-  * [Vangelis Mouhtsis](http://gnugr-blog.info/)
-  * [Kees Cook](http://outflux.net/)
+  * [John Paul Adrian Glaubitz](https://users.physik.fu-berlin.de/~glaubitz/)
```

Comment:
http://users.physik.fu-berlin.de/~glaubitz/ works

https://users.physik.fu-berlin.de/~glaubitz/ doesn't work

Reply:
Firefox warns me about an insecure site when using https://users.physik.fu-berlin.de/~glaubitz/
I will remove this link.

Reply:
Same here, should i keep the link, we get only a secure warning by firefox?

Reply:
yes, keep the link, is ok for me